lib/device_map.py

```diff
@@ -1394,7 +1394,7 @@
              'mach': 'qemu'}
 
 imx6dl_riotboard = {'device_type': 'imx6dl-riotboard',
-                    'templates': ['generic-arm-dtb-kernel-ci-boot-template.json',
+                    'templates': ['generic-uboot-tftp-ramdisk-template.jinja2',
                                   'generic-arm-dtb-kernel-ci-ltp-mm-template.json',
```

Comment:
You should remove all the `json` files as these are for v1 jobs, see how other v2 devices are configured.  Then please put in the templates list only the tests you want to run, i.e. ramdisk boot, nfs boot, kselftest, other test plans...